bindings/nodejs/rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+tab_spaces = 2
```

Comment:
**suggestion**: How about create a symlink to `rustfmt.toml` under root.

Reply:
Make sense.